src/sentry/static/sentry/app/actionCreators/streamTag.jsx

```diff
@@ -0,0 +1,29 @@
+import {Client} from '../api';
+import {t} from '../locale';
+import StreamTagStore from '../stores/streamTagStore';
+import StreamTagActions from '../actions/streamTagActions';
+import AlertActions from '../actions/alertActions';
+
+const api = new Client();
+
+const MAX_TAGS = 500;
```

Comment:
Is this constant used elsewhere? If so let's extract to `constants.jsx`.

Reply:
No, just here